pkgs/tools/system/taskspooler/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, fetchurl, makeWrapper, coreutils }:
+stdenv.mkDerivation rec {
+  pname = "taskspooler";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://vicerveza.homeunix.net/%7Eviric/wsgi-bin/hgweb.wsgi/ts/archive/a415e55ad9bd.tar.gz";
+    sha256= "08gss0pjqa84dbldlmg95rczrr9yp6sz30fbbn8wbnp93k7brzm2";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  configurePhase = ''
+    substituteInPlace Makefile \
+      --replace "PREFIX?=/usr/local" "PREFIX=$out"
+  '';
+
+  postFixup = ''
+    wrapProgram $out/bin/ts \
+      --set-default TS_SLOTS "$(${coreutils}/bin/nproc --all)"
+  '';
+
+  meta = with lib; {
+    description = "Simple single node task scheduler";
+    license = licenses.gpl2;
```

Comment:
Do you know if the license is gpl2Only or gpl2Plus?

Reply:
Just looked it up. It's gpl2Plus. Changed it.